public/javascripts/overlays/admin.js

```diff
@@ -318,7 +318,27 @@ function sendUpdate() {
         'stateRight': window.stateRight
     };
     socket.emit('update overlay', data);
-    toastNotify();
+    toastNotify('Overlay data updated.');
+}
+
+function changeLayout() {
+    var layout = document.getElementsByClassName('selectedLayout')[0];
+    var data = {
+        'layout': layout ? layout.value : 'empty-layout',
+        'background': document.getElementById('layout-background').value
```

Comment:
I don't care if it's null here. I only check in generic.hbs because I don't want the blank field on the admin page to reset the background if it is cleared somehow.